### PR TITLE
fix govern test failures

### PR DESCRIPTION
**Pull request by yuetloo**
As a temporary workaround for the coverage test failure, I took `RegisterToken.js`, `token.js`, `Gateway.js` out of the jest coverage test.

We should add them back once we consolidated the test framework to use in govern, see https://linear.app/aragon/issue/DAO-237/convert-tests-from-jest-to-mocha-in-governjs

### CODE REVIEW

File: packages/govern/public/proposal.ts [hunk 1]
```diff
@@ -65,14 +66,14 @@ export type PayloadType = {
...
-  proof: BytesLike
+  allowFailuresMap: utils.BytesLike
+  proof: utils.BytesLike
 }
 
 export type ActionType = {
   to: string
   value: BigNumberish
-  data: BytesLike
+  data: utils.BytesLike
```
Comment (novaknole) [inline packages/govern/public/proposal.ts:76]: @yuetloo why does `BytesLike` come from `utils` ? and `BigNumberish` doesn't come from it ? there're many places this happens.

**Comment by yuetloo** [comment]
Those are ethers.js types..

On May 6, 2021, at 4:50 AM, Giorgi Lagidze ***@***.***> wrote:

﻿

***@***.**** commented on this pull request.
------------------------------

In packages/govern/public/proposal.ts
<https://github.com/aragon/govern/pull/360#discussion_r627208619>:

>  }

 export type ActionType = {
   to: string
   value: BigNumberish
-  data: BytesLike
+  data: utils.BytesLike

@yuetloo <https://github.com/yuetloo> why does BytesLike come from utils ?
and BigNumberish doesn't come from it ? there're many places this happens.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/aragon/govern/pull/360#pullrequestreview-653148162>, or
unsubscribe

**Comment by novaknole** [comment]
@yuetloo I know what they are. but for bignumberish, you don't use `utils.bignumberish` . 

This is not consistent.

**Comment by yuetloo** [comment]
Doesn’t seems like bignumberish is available on utils

**Pull request merged**